erizo_controller/erizoJS/erizoJS.js

```diff
@@ -73,7 +75,10 @@ var controller = require('./erizoJSController');
 // Logger
 var log = logger.getLogger('ErizoJS');
 
-var ejsController = controller.ErizoJSController();
+var threadPool = new addon.ThreadPool(10);
```

Comment:
If I'm right, you should use here the number of workers configured in config file